storage/system-test/encryption.test.js

```diff
@@ -0,0 +1,73 @@
+// Copyright 2015-2016, Google, Inc.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+var gcloud = require('gcloud');
+var uuid = require('node-uuid');
+var storage = gcloud.storage();
+var program = require('../encryption');
+
+var bucketName = 'nodejs-docs-samples-test-' + uuid.v4();
+var fileName = 'test.txt';
+var filePath = path.join(__dirname, '../resources', fileName);
+var downloadFilePath = path.join(__dirname, '../resources/downloaded.txt');
+
+describe('storage:encryption', function () {
+  var key;
+  before(function (done) {
+    key = program.generateEncryptionKey();
+    storage.createBucket(bucketName, done);
+  });
+
+  after(function (done) {
```

Comment:
Explain what this does with a comment? (e.g. "Reset GCS bucket after running all tests" or something.)